templates/server-ingress.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.server.frontend.ingress.enabled -}}
```

Comment:
Please rename file to `frontend-ingress.yaml`.

Reply:
Thanks for the catch, renamed.

Reply:
other resources too like `service` is in `server-service.yaml`. should we really make `frontend-ingress.yaml` ?

Reply:
Yes please, I'll be adjusting file names to be more reflective as well.